internal/dataplane/sendconfig/inmemory_schema.go

```diff
@@ -10,24 +14,26 @@ import (
 type DBLessConfig struct {
 	file.Content
 	ConsumerGroupConsumerRelationships []ConsumerGroupConsumerRelationship `json:"consumer_group_consumers,omitempty"`
-	ConsumerGroupPluginRelationships   []ConsumerGroupPluginRelationship   `json:"consumer_group_plugins,omitempty"`
 }
 
 // ConsumerGroupConsumerRelationship is a relationship between a ConsumerGroup and a Consumer.
 type ConsumerGroupConsumerRelationship struct {
-	ConsumerGroup string `json:"consumer_group"`
-	Consumer      string `json:"consumer"`
+	ConsumerGroup consumerGroup `json:"consumer_group"`
```

Comment:
Do we really need to make it a nested struct? It was working fine for me when it was just a string which could be either ID or Name. That's something I'd expect to work according to the schema https://github.com/Kong/kong-ee/blob/0f110701903cc62ca037700eae1c8291ce6fd172/kong/db/schema/entities/consumer_group_consumers.lua#L4

Reply:
Fortunately you're right we don't have to do it